runtime/rasdump/dmpsup.c

```diff
@@ -60,6 +60,7 @@ char* dumpDirectoryPrefix = NULL;
 #if defined(J9ZOS390)
 #if defined(J9VM_ENV_DATA64)
 #include <__le_api.h>
+#include <ctest.h>
```

Comment:
It would help to know which part of the code in this file uses function from ctest.h

Reply:
```
openj9/runtime/rasdump/dmpsup.c:278:2: error: call to undeclared function '__cabend'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  278 |         __cabend(PORT_ABEND_CODE, PORT_ABEND_REASON_CODE, PORT_ABEND_CLEANUP_CODE);
      |         ^
```

Perhaps I can add it somewhere inside the commit description ...

Reply:
Updated commit description to reflect missing undeclared funcs for which headers were added.